lib/MusicBrainz/Server/Controller/User.pm

```diff
@@ -440,7 +440,7 @@ sub tag : Chained('load') PathPart('tag') Args(1)
     my $tag = $c->model('Tag')->get_by_name($tag_name);
     my %tags = ();
     my $tag_in_use = 0;
-    my @entities_with_tags = sort { $a <=> $b } entities_with('tags');
+    my @entities_with_tags = sort { $a cmp $b } entities_with('tags');
```

Comment:
I had considered using a collator for sorting, but the full strings (“Releases” etc.) and their translations are only available in the templates. At least it’s a consistent order now.